host-configs/lc-builds/blueos/xl_2018_10_29.cmake

```diff
@@ -0,0 +1,30 @@
+##
+## Copyright (c) 2016-18, Lawrence Livermore National Security, LLC.
+##
+## Produced at the Lawrence Livermore National Laboratory.
+##
+## LLNL-CODE-689114
+##
+## All rights reserved.
+##
+## This file is part of RAJA.
+##
+## For details about use and distribution, please read RAJA/LICENSE.
+##
+
+set(RAJA_COMPILER "RAJA_COMPILER_XLC" CACHE STRING "")
+
+set(CMAKE_CXX_COMPILER "/usr/tce/packages/xl/xl-beta-2018.10.29/bin/xlc++_r" CACHE PATH "")
```

Comment:
I don't think we need this host-config file. The 2018.11.02 one should be sufficient.